docs-ref-conceptual/azure-cli-extension-alias.md

```diff
@@ -49,27 +49,23 @@ The alias extension is under active development and new versions are released re
 az extension update --name alias
 ```
 
-## Alias commands file format
+## Manager Azure CLI aliases
 
-Alias command definitions are written into a configuration file, located at `$AZURE_USER_CONFIG/alias`. The default value of `AZURE_USER_CONFIG` is `$HOME/.azure` on macOS and Linux, and `%USERPROFILE%\.azure` on Windows. The alias configuration file is written in the INI configuration file format. The general format for alias commands is:
+The alias extension provides convenient and familiar commands to manage aliases. To view all the available commands and parameter detail, invoke the alias command with the help flag.
 
-```
-[command_name]
-command = invoked_commands
+```azurecli
+az alias --help
 ```
 
-Don't include `az` as part of the command.
+## Examples
```

Comment:
Remove this H2. H2s in our docs are always fully fleshed out sections that are designed to be easy to find and navigate to from the right-hand ToC. Users are more likely to look for a specific _type_ of example rather than just _any_ example.